Dockerfile

```diff
@@ -20,11 +20,11 @@ WORKDIR /app
 # Install latest chrome dev package libs so that the bundled version of Chromium installed by Puppeteer will work
 # https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-in-docker
 RUN apt-get update \
-    && apt-get install -y wget gnupg gnupg1 gnupg2 make python \
```

Comment:
slightly concerned as thought python was a dependency of the logging framework but I assume it's been tested and I'm probably wrong

Reply:
I'll check the logging.  Do you know why it is needed? Is there something I can read?